samza-api/src/main/java/org/apache/samza/runtime/ApplicationRunner.java

```diff
@@ -35,7 +35,7 @@
 @InterfaceStability.Unstable
 public interface ApplicationRunner {
 
-  String RUNNER_CONFIG = "job.runner.class";
+  String RUNNER_CONFIG = "app.runner.class";
```

Comment:
Issue: Did we decide to rename this? If not yet, let's get feedback from @nickpan47 @jmakes @navina etc.